homeassistant/strings.json

```diff
@@ -45,7 +45,8 @@
         "latitude": "Latitude",
         "location": "Location",
         "pin": "PIN Code",
-        "mode": "Mode"
+        "mode": "Mode",
+        "country_code": "Country Code"
```

Comment:
IMO, this should be in a separate PR as you are refactoring something this does not pertain to the PR title.

Reply:
It was only due to a failing style check that I added it here. But to be honest I don't know the policy on adding entries here. It seems that every config option used in any config flow needs an entry here?

Reply:
This should be done in a new PR, and it should update all existing places that refer to a country code, and it should migrate one existing entry to pre-populate this field. 

So yes, for now revert this please.